manager/integration/tests/test_recurring_job.py

```diff
@@ -259,12 +259,11 @@ def test_recurring_job(set_random_backupstore, client, volume_name):  # NOQA
         elif "backup2-" in b.snapshot:
             complete_backup_2_count += 1
 
-    # 2 completed backups from backup1
-    # 2 or more completed backups from backup2
+    # 1 completed backups from backup1
+    # 2 completed backups from backup2
     # NOTE: NFS backup can be slow sometimes and error prone
-    assert complete_backup_1_count == 2
-    assert complete_backup_2_count >= 2
-    assert complete_backup_2_count < 4
```

Comment:
Could you explain why we had these assertions before?

Reply:
Maybe @c3y1huang know the reason.

Reply:
ya, forgot the details, but it seems like a timing issue so we made this tolerance for the counts. 

@chanow816 But why https://github.com/longhorn/longhorn/issues/3035 can fix this issue if the timing problem still exists?